MSAL/IdentityCore

```diff
@@ -1 +1 @@
-Subproject commit 8aef53794c6586567fe0af656193601d7e9db540
```

Comment:
This pull request does not update CHANGELOG.md.

Please consider if this change would be noticeable to a partner or user and either update CHANGELOG.md or resolve this conversation.
<!-- changelog-comment -->